core/database/api/data_router.js

```diff
@@ -0,0 +1,1362 @@
+'use strict';
+
+const   createRouter = require('@arangodb/foxx/router');
+const   router = createRouter();
+const   joi = require('joi');
+const   g_db = require('@arangodb').db;
+const   g_lib = require('./support');
+const   g_proc = require('./process');
+const   g_tasks = require('./tasks');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ORNL/DataFed2/pullRequest?prid=8791729)
```suggestion
const   g_tasks = require("./tasks");
```